uint16 onTime
uint16 offTime
int8 targetCount
uint16 lastState
uint16 update
int8 order